versioned_docs/version-V3/guides/providing-liquidity/mint-a-new-position.md

```diff
@@ -0,0 +1,152 @@
+---
+id: mint-a-position
+title: Mint a New Position
+sidebar_position: 2
+---
+
+## Input Parameters
+
+To mint a new position, we use the `nonFungiblePositionManager` and call `mint`.
+
+For the sake of this example, were hard coding the token amounts to be minted, in production this would be a user configurable amount
```

Comment:
For the sake of this example, we're hard coding the token amounts to be minted. In production this would be a user configurable amount defined as an input into the function.